src/control/lib/control/pool.go

```diff
@@ -428,9 +428,9 @@ type (
 
 	// PoolQueryResp contains the pool query response.
 	PoolQueryResp struct {
-		Status int32  `json:"status"`
-		State  string `json:"state"`
-		UUID   string `json:"uuid"`
+		Status int32                   `json:"status"`
+		State  system.PoolServiceState `json:"state"`
```

Comment:
Great thanks for fixing this, please ignore the earlier comment about it as I'm having to go through the commits individually because on the PR there was no way to view changes since my last review.